.github/workflows/doc-check.yml

```
@@ -52,17 +52,17 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-          language: [en,es,zh_Hans]
+          language: [en,es,zh_Hans,sv]
```

YAMLlint (1.37.1)

[error] too few spaces after comma

(commas)